tests/unit/preprocessor/_time/test_time.py

```diff
@@ -1022,6 +1022,19 @@ def test_standardized_anomalies(period, standardize=True):
             )
 
 
+@pytest.mark.parametrize('period, reference', PARAMETERS)
+def test_anomalies_preserve_metadata(period, reference, standardize=False):
+    cube = make_map_data(number_years=2)
+    cube.var_name = "si"
+    cube.units = "m"
+    result = anomalies(cube, period, reference, standardize=standardize)
+    assert result.metadata == cube.metadata
+    for coord_cube, coord_res in zip(cube.coords(), result.coords()):
```

Comment:
Note that if one array is longer than the other, zip will only iterate over the number of elements in the shortest array, so this does not test that the cubes have the same coordinates

Reply:
good point actually! but the cubes **should** have the same number of coords otherwise the iris cube difference would fail beforehand